twitter_account.rb

```diff
@@ -0,0 +1,10 @@
+require 'active_record'
```

Comment:
[Q] Do we really need active_record?

Reply:
[IMO] Less dependencies would be great here 💪

Reply:
Thank you for reply. I search 💪